### PR TITLE
Fix: Sort list of tables in table dropdown

### PR DESCRIPTION
**Pull request by colemanProjects**
This allows for an alphabetic sort for the tables dropdown.

### CODE REVIEW

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import { test, moduleForComponent } from 'ember-qunit';
+
+moduleForComponent('report-builder', 'Unit | Component | Report Builder', {
+  unit: true,
+  needs: [
+    'component:report-builder',
```
Comment (balajimanoharan) [inline packages/reports/tests/unit/components/report-builder-test.js]: This should be pulled in by the `moduleForComponent`. So we don't need to `need` this.

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    'service:bard-metadata'
+  ]
+});
+
+test('allTables', function(assert){
+  assert.expect(1);
+
+  let component = Ember.run(() => this.subject()),
```
Comment (balajimanoharan) [inline packages/reports/tests/unit/components/report-builder-test.js]: Can we do `import { run } from '@ember/runloop'`?

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      {longName: "12345", id: '2'},
+      {longName: "9876", id: '1'},
+      {longName: "advertisement", id: '8'},
+      {longName: "Advertisement", id: '7'},
+      {longName: "DATASOURCE_A", id: '4'},
+      {longName: "DATASOURCE_B", id: '3'},
+      {longName: "table-A", id: '6'},
+      {longName: "table-B", id: '5'}
+    ];
+  Ember.set(component, 'metadataService', { all: function(option) {
```
Comment (balajimanoharan) [inline packages/reports/tests/unit/components/report-builder-test.js]: new import syntax
`import { get, set } from '@ember/object'`?

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    return Ember.A([
```
Comment (balajimanoharan) [inline packages/reports/tests/unit/components/report-builder-test.js]: `import { A as arr } from '@ember/array'`?

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
```
Comment (balajimanoharan) [inline packages/reports/tests/unit/components/report-builder-test.js]: You shouldn't need this anymore.

File: packages/reports/tests/unit/components/report-builder-test.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        {longName: "12345", id: '2'},
+        {longName: "9876", id: '1'},
+        {longName: "advertisement", id: '8'},
+        {longName: "Advertisement", id: '7'},
+        {longName: "DATASOURCE_A", id: '4'},
+        {longName: "DATASOURCE_B", id: '3'},
+        {longName: "table-A", id: '6'},
+        {longName: "table-B", id: '5'}
+      ];
+  set(component, 'metadataService', { all: () => arr([
```
Comment (jkusa) [inline packages/reports/tests/unit/components/report-builder-test.js]: Can we put `all: () => arr([` on the next line?

**Pull request merged**